src/components/Login.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import '../style/Login.css';
+import { useHistory } from 'react-router-dom';
+import Logo from '../common/Logo';
+import TweetBtn from '../common/TweetBtn';
+import { login } from '../actions/loginActions'
+import { connect } from 'react-redux';
+
+
+function Login(props) {
+    const [name, setName] = useState('');
+    const [password, setPassword] = useState('');
+    const history = useHistory();
+
+    function handleNameChange(e) {
+        setName(e.target.value);
+    }
+    function handlePasswordChange(e) {
+        setPassword(e.target.value);
+    }
```

Comment:
If you had 10 fields in the form, how would you implement it?
Hint: read in React docs about controlled components and will see how to implement it in a generic way